### ISSUE TITLE
Consideration of charging location possible?

### ISSUE THREAD

**Issue by fabmid** [RAMP-project/RAMP-mobility#35]
Hi,
I would like to use RAMP-mobility for generating charging profiles for a vehicle fleet of a neighborhood (~100 cars).
In this context it is relevant for me where the charging process occurs (at home in the neighborhood or not at home) to see the impact of vehicle charging on the local neighborhood power system.

Is it possible to consider charging location within the current RAMP-mobility version?
Thanks!

**Comment by FLomb** [comment]
Hi,

As you correctly noted, as of now, the model does not explicitly distinguish between car locations. The user/car behaviour is a function of the time of the day and outdoor conditions. For example, the "night charging" strategy is guided by simple time-driven data (based on time-of-use surveys) about when users are at home after the end of their work/study/other activities in the evening. There is no explicit data about car locations underlying the model, just about users' activity patterns. This is, at the same time, a strength (as it requires fewer data and it's simple to run compared to Markov-chain counterparts) and a limitation of the model (since it might become more tricky to apply it to research questions for which the exact car position throughout the day is critical).

To summarise, the model as is might allow you to have a rough understanding of when cars are at home at the end of the day and until the morning, as far as concerns working/studying users who are not at home during the day. For those users, instead, that are at home during the day, it might be a bit more complicated to tell exactly where the car is throughout the day. I guess one could extrapolate the information based on travel patterns. 

It could be an interesting area of development for us to test if we could still provide some approximate information in this sense based on our current model design and how precise that would be. It's indeed not the first time that the question gets asked; thanks for opening an issue!
